sdcm/tester.py

```diff
@@ -1132,8 +1133,10 @@ def create_table(self, name, key_type="varchar",  # pylint: disable=too-many-arg
             # will default to lz4 compression
             query += ' AND compression = {}'
 
-        if compaction is not None:
-            query += " AND compaction={'class': '%s'}" % compaction
+        if compaction is not None or sstable_size:
+            compaction = compaction or self.params.get('compaction_strategy', default=CompactionStrategy.SIZE_TIERED.value)
+            sstable_size = sstable_size or self.params.get('sstable_size', default=DEFAULT_SSTABLE_SIZE)
+            query += " AND compaction={'class': '%s', 'sstable_size_in_mb' : '%s'}" % (compaction, sstable_size)
```

Comment:
This parameter is valid for all compactions strategies ?

Reply:
no it's not. like always, user has to know what he's doing.

Reply:
so this code doesn't take that into account, i.e. it always passes this variable.
please change it to accordingly, also if it's None, this code would fail.

Reply:
I think that this param is relevant for all compactions strategies.
I wouldn't deal with the compactions strategy at all, only with this file size at the moment (as your task suggest).
The default of scylla is SizeTier and wherever we set it through c-s, we usually don't use pre-create schema.

Reply:
code is changed to support all possible combinations in any case.